src/Esfa.Vacancy.Register.Application/Esfa.Vacancy.Register.Application.csproj

```diff
@@ -65,17 +65,23 @@
   <ItemGroup>
     <Compile Include="Exceptions\UnauthorisedException.cs" />
     <Compile Include="Exceptions\ResourceNotFoundException.cs" />
+    <Compile Include="Interfaces\IFrameworkCodeConverter.cs" />
```

Comment:
The Converter interfaces can be inside the SearchAppVac folder itself. The interfaces folder is used for anything that is shared across like the services which are implemented in the Infrastructure layer. This way we keep Application layer as cohesive as possible.
FYI, the service interfaces are defined in application layer as they are consumed in application and application is an inner layer and shouldn't know anything about outer layer (infra).